### PR TITLE
[WIP] cleanup linux build process

### PR DESCRIPTION
**Pull request by lambdalemon**
This might break the windows build?

appimagetool is not working, probably because appimage requires relative not absolute path and the qml files are specified in absolute path? Might be fixable with qrc?

Also has to disable the togglebox indicator because it's not working...

### CODE REVIEW

File: client_overlay/bin/x64/res/qml/MyToggleButton.qml [hunk 1]
```diff
@@ -7,24 +7,24 @@ CheckBox {
...
-		x: parent.leftPadding
-		y: parent.height / 2 - height / 2
-		color: parent.activeFocus ? myPalette.mid : myPalette.base
-		border.width: 0
-		Image {
-			width: 38
-			height: 38
-			x: (parent.width - width) / 2
-			y: (parent.height - height) / 2
-			source: "image://default/check/#ffffff"
```
Comment (pottedmeat7) [inline client_overlay/bin/x64/res/qml/MyToggleButton.qml:22]: These elements come from the initial parent project https://github.com/matzman666/OpenVR-InputEmulator 
I have had to make a few adjustments with newer QT versions etc, but it mostly works on Windows pretty well, Linux seems to have a few more conflicts with some of the QT elements

**Review by pottedmeat7 — APPROVED** [review]
This LGTM, doesn't look like it should affect the windows install too much other than `WatchdogProvider` I forget if a certain service uses these paths during certain Driver DLL_EXPORT loading steps maybe.. 
https://github.com/pottedmeat7/OpenVR-WalkInPlace/pull/93/files#diff-5093893db60675e2938b3442c39149eb7176070aaea3ae5d64a430823ab415ddL22-L25

**Comment by lambdalemon** [comment]
Could that be something leftover from a previous version of openvr sdk? I was referencing the newest driver tutorial https://github.com/ValveSoftware/openvr/blob/master/samples/drivers/drivers/tutorial/README.md and `WatchdogProvider` does not seem to be required.

Do I need to modify the `.sln` file to make sure everything still works on windows? Or would that belong to a different commit?

**Comment by pottedmeat7** [comment]
It could be old and not necessary by now. Yeah getting the SLN updated and windows running can be another following commit. I'll probably get to it soon and then I can create a newer installer also.

**Pull request merged**